### PR TITLE
Add Python 3 compatibility

### PR DESCRIPTION
**Pull request by WojtekKowaluk**
Also drop Python 2.5 compatibility (as suggested by Pekka)

### CODE REVIEW

File: src/main/python/RemoteSwingLibrary.py [hunk 1]
```diff
@@ -298,10 +302,14 @@ def set_java_tool_options(self):
...
         t = tempfile.NamedTemporaryFile(prefix='grant_all_', suffix='.policy', delete=False)
-        t.write("""
+        text = """
             grant {
                 permission java.security.AllPermission;
-            };""")
+            };
+            """
+        if sys.version_info[0] >= 3:
+            text = bytes(text, "UTF-8")
```
Comment (pekkaklarck) [inline src/main/python/RemoteSwingLibrary.py]: Could the above be just `b"""grant ..."""`? The `b` prefix makes it bytes in Python 3 and is a no-op on Python 2.

As a general note, could the code use `with tempfile.NamedTemporaryFile(...) as t` approach?

**Comment by pekkaklarck** [comment]
Code looks very good. I haven't tested this, though. Does @jussimalinen have time to look at this?

File: src/main/python/RemoteSwingLibrary.py (hunk 1 above)
Comment (WojtekKowaluk) [inline src/main/python/RemoteSwingLibrary.py]: Thanks, changed.

**Comment by jussimalinen** [comment]
Brilliant stuff! Thanks!

**Comment by jussimalinen** [comment]
Could you guys take a look at cleaning up the PR from #35 a bit so that we would get an easy way to run the tests on python3? You dont need to do everything I mentioned, but atleast fixing the bug with gettting a dev version has to be fixed before a merge.

**Comment by stasiek** [comment]
I cleaned it up a bit, but my company computer fried today, so I'm still waiting till tomorrow to test it on windows.

**Pull request merged**